migration_scripts/to_version_7_1/index.js

```diff
@@ -0,0 +1,118 @@
+/*
+ *    Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+const libphonenumber = require('libphonenumber-js/max');
+
+// Update the following credentials before running the script
+const DB_HOST = "";
+const DB_USER = "";
+const DB_PASSWORD = "";
+const DB_NAME = "";
+const CLIENT = ""; // Use "pg" for PostgreSQL and "mysql2" for MySQL DB
+
+if(!DB_HOST || !CLIENT) {
+  console.error('Please update the DB_HOST, DB_USER, DB_PASSWORD, DB_DATABASE and CLIENT variables before running the script.');
+  return;
+}
+
+const knex = require('knex')({
```

Comment:
how many db connections did this script use? I assume just one?

Reply:
Yeah it uses just 1. The default pool config was { min: 2, max: 10 }. I have changed that to a more conservative value of  { min: 0, max: 5 }.

Reply:
We should try and use minimal number of connections since it's happening in a live db. Can we just set it to min: 1, max: 1 so that it uses just one connection for the whole script?

Reply:
Updating `passwordless_devices` table is really fast regardless of connection pool size. But for `passwordless_users` and `passworldless_user_to_tenant` tables the script on average takes - 

```
4500ms per 1000 rows with MAX 10 connections
4700ms per 1000 rows with MAX 5 connections
19000ms per 1000 rows with MAX 1 connections
```

Seeing this, I believe the default `MAX_POOL_SIZE` should be 5. I have also updated the changelog informing user that they can increase this value to potentially increase the speed even more.

Reply:
Cool